SwiftReflector/NewClassCompiler.cs

```diff
@@ -4937,9 +4937,12 @@ static bool TypesMatch (FunctionDeclaration parentDecl, NamedTypeSpec spec, Swif
 			if (prop.TLFGetter != null) {
 				var marshaler = new MarshalEngine (use, useLocals, TypeMapper, wrapper.Module.SwiftCompilerVersion);
 
-				var getter = TLFCompiler.CompileMethod (getterWrapperFunc, use, null, null, getterName, false, false, true);
-				var thisParm = getter.Parameters [0];
-				getter.Parameters [0] = new CSParameter (thisParm.CSType, thisParm.Name, CSParameterKind.This);
+				var wrapperGetter = TLFCompiler.CompileMethod (getterWrapperFunc, use, null, null, getterName, false, false, true);
```

Comment:
Side note: it's getting hard (the diff) to review with so many `null` and boolean arguments.
Maybe you can start to use named arguments ?
Not sure it would help you (you know the calls) but reviewers and possible contributors would have an easier time.

Reply:
👍 https://github.com/xamarin/binding-tools-for-swift/issues/378